context.go

```diff
@@ -344,6 +344,9 @@ func (c *Context[H]) CreatePreBlock() PreBlock[H] {
 		if c.preBlock = c.MakePreHeader(); c.preBlock == nil {
 			return nil
 		}
+		if !c.hasAllTransactions() {
```

Comment:
> it's a dead code

It's not, because there's one place where we call this possibly missing transactions. I refactored this place in the last commit.

Reply:
Seems like I've been looking at the old version that doesn't have 3b0a52287b5ba8853191b51b758b81b6266190f7.

Reply:
https://github.com/nspcc-dev/dbft/commit/f2e6bd26f42d51ae1d1bb5eca6c81d4c5a50bd9e is a bugfix, we have a broken library without it anyway.